app/dashboards/user_dashboard.rb

```diff
@@ -25,13 +27,15 @@ class UserDashboard < Administrate::BaseDashboard
   COLLECTION_ATTRIBUTES = %i[
     id
     email
-    confirmed_at
-    created_at
+    matches
+    receivers
```

Comment:
For the same reason as mentioned below, these shouldn't be here.

Also, why remove the `confirmed_at` and `created_at` fields? I find the `confirmed_at` quite handy when checking if a certain user has been confirmed.

Reply:
the main reason was due to the clutter in the page display but also I assumed that the redeem message will be done automatically and we would not need to check this information. But can be handy the confirmed_at information, the created_at I do not see much utility on this page. Will add back the created_at field

Reply:
Yeah, agree. The `confirmed_at` should at least be there. The `created_at`, meh, not so useful 😛